sympy/physics/mechanics/tests/test_kane3.py

```diff
@@ -253,7 +255,9 @@ def test_bicycle():
     # many rows as *total* coordinates and speeds, but only as many columns as
     # independent coordinates and speeds.
 
-    forcing_lin = KM.linearize()[0]
+    with warnings.catch_warnings():
+        warnings.filterwarnings('always')
```

Comment:
Python [library](https://docs.python.org/2/library/warnings.html#temporarily-suppressing-warnings) seems to recommend ``simplefilter`` as there is no message included.